src/components/ContentWrapper.jsx

```diff
@@ -303,49 +330,31 @@ const ContentWrapper = (props) => {
         backendLoading || !backendStatusRequested) {
         return <PreloadContent />;
       }
-
-      if (backendError) {
-        return <Error errorText={backendError} />;
-      }
-
-      if (gem2sRunningError) {
-        return <GEM2SLoadingScreen experimentId={routeExperimentId} pipelineStatus='error' pipelineType='gem2s' />;
+      if (currentStatusScreen && currentStatusScreen.type !== 'qc') {
```

Comment:
any reason not to use the `?` operator here like below?
```suggestion
      if (currentStatusScreen?.type !== 'qc') {
```

Reply:
because when you dont have a status screen (its succeeded) `currentStatusScreen` will be null, but still go into this if